### PR TITLE
Remove "sex" and "sexual"

### PR DESCRIPTION
**Pull request by zedthehedgehog**
Since "ass" and "damn" have been removed, these should be removed too. They're not that bad. This also unintentionally censors "Bisexual".

### CODE REVIEW

File: better_profanity/profanity_wordlist.txt [hunk 1]
```diff
@@ -591,8 +591,6 @@ scrot
 scrote
 scrotum
 semen
-sex
-sexual
 shamedame
 shemale
 shibari
```
Comment (MikeDev101) [inline better_profanity/profanity_wordlist.txt:596]: ouch my search history

**Review by MikeDev101 — CHANGES_REQUESTED** [review]

**Comment by MikeDev101** [comment]
Also, trying to clean up inactive branches

**Comment by JoshAtticus** [comment]
> Since "ass" and "damn" have been removed, these should be removed too. They're not that bad. This also unintentionally censors "Bisexual".

Late but this is a bit more hardcore than ass or damn

**Comment by theotherhades** [comment]
tbh sex and sexual are more likely to be used in a non-dirty context than ass, so if that's unblocked idk why sex and sexual shouldn't be

**Comment by tnix100** [comment]
Sexual should not be blocked imo, but then sex would probably need to be unblocked as well. As @theotherhades said, they're less likely to be used in a dirty context than ass.

**Comment by JoshAtticus** [comment]
> Sexual should not be blocked imo, but then sex would probably need to be unblocked as well. As @theotherhades said, they're less likely to be used in a dirty context than ass.

Well then ass should be reblocked because no one uses ass to describe a donkey.

**Comment by MikeDev101** [comment]
> Well then ass should be reblocked because no one uses ass to describe a donkey.

Except for Shrek.

**Comment by mdwalters** [comment]
shrek

**Comment by MikeDev101** [comment]
Closing because of going stale, and that we've modified things to move BF away from this repo.

**Pull request closed without merging**